apps/coscrad-frontend/src/configurable-front-matter/getConfigurableContent.ts

```diff
@@ -1,14 +1,14 @@
 import { ConfigurableContent } from './data/configurableContentSchema';
-import rawData from './data/content.config.json';
+import { contentConfig } from './data/content.config';
 import { InvalidContentConfigurationException } from './errorHandling/exceptions/InvalidContentConfigurationException';
 import { validateConfigurableContent } from './validation/validateConfigurableContent';
 
 export const getConfigurableContent = (): ConfigurableContent => {
-    const validationResult = validateConfigurableContent(rawData);
+    const validationResult = validateConfigurableContent(contentConfig);
 
     if (validationResult.length > 0) {
```

Comment:
We need to validate all props and test this behaviour before merging in.

Reply:
I decided to make that a [separate story](https://www.pivotaltracker.com/story/show/184076888).